catroid/res/values/strings.xml

```diff
@@ -561,7 +560,11 @@
     <string name="error_drone_wrong_platform">We can not run a ARDrone project on your device.</string>
 	<string name="error_rename_incompatible_project">Renaming an incompatible project isn\'t possible.</string>
     <string name="error_changing_description_of_incompatible_project">Changing the description of an incompatible project isn\'t possible.</string>
-    <string name="error_unknown_error">An unknown error occurred.</string>
+    <string name="error_malformed_url">Something was wrong with the URL for the Play Store.</string>
+    <string name="error_IOException_html">Something went wrong, when downloading the current version Number.\n Error Code: HTML_DOWNLOAD_ERROR</string>
+    <string name="error_IOException_regex_one">Something went wrong, when downloading the current version Number.\n Error Code: REGEX_ERROR_ONE</string>
+    <string name="error_IOException_regex_two">Something went wrong, when downloading the current version Number.\n Error Code: REGEX_ERROR_TWO</string>
```

Comment:
I think something like `error_io_exception_regex_one` would be more consistent with the other string names.

Also, I'd write "Something went wrong while downloading,..." without the comma and "while" instead of "when". Or "during the download of". And "number" instead of "Number".